CHANGELOG.md

```diff
@@ -80,4 +80,4 @@ v3.1 requires Elixir v1.5+.
 
 ## v3.0.0 (2018-10-29)
 
-  * Initial release
\ No newline at end of file
+  * Initial release
```

Comment:
Don't know why this shows as a change. (maybe some side effect from the GitHub UI). I can manually remove it if wanted?

Reply:
Missing trailing line at the end of the file. GitHub is correct here. It is generally better to have one. :)